### PR TITLE
Refactoring command line

### LINKED ISSUE #80: Remove `--baseline` flag from `tsvtool analysis`
Automatically detect which files are available to conduct the analysis.

### LINKED ISSUE #82: Remove tsv_path from mandatory arguments of generate
Generate could detect automatically the subjects and sessions that could be used to synthetize new data.
Eventually we could let it as an option if the user wants to specify which part of the CAPS they want to use (but I don't think this is really useful...)

### PR DESCRIPTION
**Pull request by 14thibea**
This PR closes #82 and partly answers #52.

### CODE REVIEW

**Comment by 14thibea** [comment]
Last commit closes #80.

**Review by mdiazmel — COMMENTED** [review]
Hi @14thibea ,
Seems ready to merge. Just one detail, the PR modify the output of the command line and documentation needs also to be edited.  Could you please update the files in the `AD-DL/docs` folder?
Thanks,

**Comment by 14thibea** [comment]
Hi @mdiazmel ,
Would it be possible to close first #101 that takes into account commits which were already merged? And then I will complete in this PR to add the new changes.

**Comment by mdiazmel** [comment]
> Would it be possible to close first #101 that takes into account commits which were already merged? And then I will complete in this PR to add the new changes.

I just closed PR #101!

File: docs/Installation.md [hunk 1]
```diff
@@ -6,70 +6,53 @@ Please do not hesitate to contact us on the
...
 bash /tmp/miniconda-installer.sh
 ```
 
-#### Installation of clinicadl
-
+## Install ClinicaDL
 
-We recommend to use `conda` or `virtualenv` to  install clinicadl inside a
-python environment. E.g.,
+The latest release of Clinica can be installed using `pip` as follows:
```
Comment (mdiazmel) [inline docs/Installation.md]:
```suggestion
The latest release of ClinicaDL can be installed using `pip` as follows:
```

**Review by mdiazmel — COMMENTED** [review]
Thanks @14thibea 
Ready to merge!

**Pull request merged**